include/net/nrf_cloud_cell_pos.h

```diff
@@ -36,6 +36,20 @@ struct nrf_cloud_cell_pos_result {
 	uint32_t unc;
 };
 
+/** @defgroup nrf_cloud_cell_pos_omit Omit item from cellular positioning request.
+ * @{
+ */
+#define NRF_CLOUD_CELL_POS_OMIT_TIME_ADV	LTE_LC_CELL_TIMING_ADVANCE_INVALID
+#define NRF_CLOUD_CELL_POS_OMIT_RSRQ		LTE_LC_CELL_RSRQ_INVALID
+#define NRF_CLOUD_CELL_POS_OMIT_RSRP		LTE_LC_CELL_RSRP_INVALID
+#define NRF_CLOUD_CELL_POS_OMIT_EARFCN		(LTE_LC_CELL_EARFNC_MAX + 1)
```

Comment:
Because the EARFCN range can/will change, would it better to set `NRF_CLOUD_CELL_POS_OMIT_EARFCN` to `UINT32_MAX` instead?

Reply:
sounds good 👍